winml/lib/Common/inc/iengine.h

```diff
@@ -66,6 +66,9 @@ IModelInfo : IUnknown {
   STDMETHOD(GetName)
   (const char** out, size_t* len) PURE;
 
+  STDMETHOD(SetName)
+  (const char* name) PURE;
```

Comment:
is the convention that we expect the char* to be null terminated and therefore we don't need to pass the length?

Reply:
Yes, I assume the char* will be null terminated